.github/workflows/build.yml

```
@@ -8,4 +8,4 @@ jobs:
     - uses: cachix/install-nix-action@v27
       with:
         nix_path: nixpkgs=channel:nixos-24.05
-    - run: nix --experimental-features "nix-command flakes" build
+    - run: nix --experimental-features "nix-command flakes" check
```

```suggestion
    - run: nix --experimental-features "nix-command flakes" flake check
```
This is probably what's breaking it, not sure thou :woman_shrugging:

---

took the liberty of pushing that fix